### DISCUSSION TITLE
Any other way to set developer team for tauri ios?

### DISCUSSION THREAD

**Discussion by nooblyf** [tauri-apps/tauri#8325] [Q&A] [answered]
I read previous discussion where someone said to put any bogus value in `tauri.conf.json` like this

```json
"tauri": { "bundle": { "iOS": { "developmentTeam": "12345"}}}
```

but i don't want to commit this to a public repo, not sure if it is safe to do so?
is there any other way, i tried setting the `APPLE_DEVELOPMENT_TEAM` env variable but it doesn't run in dev mode

i did it like this
```sh
$ APPLE_DEVELOPMENT_TEAM="12345" pnpm tauri ios init
$ APPLE_DEVELOPMENT_TEAM="12345" pnpm tauri ios dev
```

even tried exporting
```sh
$ export APPLE_DEVELOPMENT_TEAM=12345
```

i'm on `Mac OS 14.1.2 X64`

**Comment by FabianLars** [comment, marked as answer]
> but i don't want to commit this to a public repo, not sure if it is safe to do so?

Why wouldn't it be? Assuming you're setting a bogus value why would it matter? But even if it's the actual value of your dev team, afaik there are no real risks with that (or it wouldn't be in the config).

That said, the env var you showed should do the same. If you're on an older tauri version is may still be called `TAURI_APPLE_DEVELOPMENT_TEAM` though

**Reply by nooblyf** [reply]
> Why wouldn't it be? Assuming you're setting a bogus value why would it matter? But even if it's the actual value of your dev team, afaik there are no real risks with that (or it wouldn't be in the config).

i don't know, i just assumed if it's a key it must be hidden? i didn't make it clear but i was asking for when i have to add my real key, but i'm glad that there aren't any risks with just putting it in `tauri.conf.json`, thanks!
